.github/workflows/release.yml

```diff
@@ -22,21 +22,20 @@ jobs:
         uses: gradle/actions/setup-gradle@v3
       - name: Run build with Gradle Wrapper
         run: ./gradlew build
-      - name: Bump version and push tag
-        id: tag_version
-        uses: mathieudutour/github-tag-action@v6.1
-        with:
-          release_branches: "release"
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Create a GitHub release
+      - name: Tag version 3.1.0
+        run: |
+          git tag 3.1.0
```

Comment:
@cameronvoell thoughts on versioning here? I could also be down with `3.0.29-patch1` or something?

Reply:
Also going to have to update the release script to pull from this branch instead of main to make this happen.